builtin/providers/aws/resource_aws_rds_cluster_test.go

```diff
@@ -17,15 +17,65 @@ import (
 func TestAccAWSRDSCluster_basic(t *testing.T) {
 	var v rds.DBCluster
 
+	ri := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	config := fmt.Sprintf(testAccAWSClusterConfig, ri)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSClusterDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSClusterExists("aws_rds_cluster.default", &v),
+					resource.TestCheckResourceAttr(
+						"aws_rds_cluster.default", "preferred_backup_window", "07:00-09:00"),
```

Comment:
Let's separate these into their own test, `TestAccAWSRDSCluster_backups` or something. The theme I'd like to keep with the `_basic` tests is that they should represent the bare minimum configuration. Doing so would have revealed the `panic` regarding `preferred_maintenance_window` I mention above :smile: 

Once we start testing and checking changes like you do below, with the pre and post configs, we're no longer in `basic` territory